src/tasks/batch_integration/methods/scanvi/script.py

```diff
@@ -1,30 +1,74 @@
-import yaml
 import anndata as ad
-from scib.integration import scanvi
+from scvi.model import SCVI, SCANVI
 
 ## VIASH START
 par = {
-    'input': 'resources_test/batch_integration/pancreas/unintegrated.h5ad',
+    'input': 'resources_test/batch_integration/pancreas/dataset.h5ad',
     'output': 'output.h5ad',
-    'hvg': True,
+    'n_hvg': 2000,
+    'n_latent': 30,
+    'n_hidden': 128,
+    'n_layers': 2,
+    'max_epochs_scvi': 400,
+    'max_epochs_scanvi': 400
 }
 meta = {
-    'functionality_name' : 'foo',
-    'config': 'bar'
+    'functionality_name' : 'scanvi',
 }
 ## VIASH END
 
-
-
 print('Read input', flush=True)
 adata = ad.read_h5ad(par['input'])
 
+if par["n_hvg"]:
+    print(f"Select top {par['n_hvg']} high variable genes", flush=True)
+    idx = adata.var["hvg_score"].to_numpy().argsort()[::-1][:par["n_hvg"]]
+    adata = adata[:, idx].copy()
+
+# based on scib
+# -> https://github.com/theislab/scib/blob/main/scib/integration.py#L290-L297
+if not par["max_epochs_scvi"]:
+    par["max_epochs_scvi"] = min(int(round(20000 / adata.n_obs) * 400), 400)
```

Comment:
nit: we apply this heuristic [here](https://github.com/scverse/scvi-tools/blob/main/scvi/model/_utils.py#L35-L59) so technically this is not needed